server/index.js

```diff
@@ -95,6 +95,7 @@ global.LoginServer = require('../.server-dist/loginserver').LoginServer;
 global.Ladders = require('./ladders');
 
 global.Chat = require('./chat');
+global.__version = {head: Chat.version()};
```

Comment:
I would put this in `Chat` instead of introducing another global.

Reply:
it would be erased on `/hotpatch chat`

Reply:
Also, I didn't introduce a new global, this global already exists, its just only set in room-battle and battle.ts.

Reply:
Oh, fair, I guess this makes sense, then.